spatialdata/_core/_spatialdata.py

```diff
@@ -290,12 +290,15 @@ def _locate_spatial_element(self, element: SpatialElement) -> tuple[str, str]:
             raise ValueError("Element found multiple times in the SpatialData object.")
```

Comment:
could this line print the element types and names?

Reply:
done, and added tests for the `_locate_spatial_element()` method.